src/main/java/org/broadinstitute/consent/http/db/DataAccessRequestDAO.java

```diff
@@ -13,100 +15,165 @@
 import org.jdbi.v3.sqlobject.statement.SqlUpdate;
 import org.jdbi.v3.sqlobject.transaction.Transactional;
 
-import java.util.List;
-
 /**
- * For all json queries, note the double `??` for jdbi3 escaped jsonb operators: https://jdbi.org/#_postgresql
+ * For all json queries, note the double `??` for jdbi3 escaped jsonb operators:
+ * https://jdbi.org/#_postgresql
  */
 @SuppressWarnings({"SqlResolve", "SqlNoDataSourceInspection"})
 @RegisterRowMapper(DataAccessRequestMapper.class)
 public interface DataAccessRequestDAO extends Transactional<DataAccessRequestDAO> {
 
-    /**
-     * Find all non-draft/partial DataAccessRequests
-     * @return List<DataAccessRequest>
-     */
-    @SqlQuery("SELECT id, reference_id, draft, (data #>> '{}')::jsonb AS data FROM data_access_request " +
-            "  WHERE not (data #>> '{}')::jsonb ??| array['partial_dar_code', 'partialDarCode'] " +
-            "  AND draft != true ")
-    List<DataAccessRequest> findAllDataAccessRequests();
+  /**
+   * Find all non-draft/partial DataAccessRequests
+   *
+   * @return List<DataAccessRequest>
+   */
+  @SqlQuery(
+      "SELECT id, reference_id, draft, user_id, create_date, sort_date, submission_date, update_date, (data #>> '{}')::jsonb AS data FROM data_access_request "
+          + "  WHERE not (data #>> '{}')::jsonb ??| array['partial_dar_code', 'partialDarCode'] "
```

Comment:
Just asking for clarification, is the WHERE statement  looking for records whose data column does not contain either ```partial_dar_code``` or ```partialDarCode``` in the jsonb data?

Reply:
Exactly. Now that I look at this, I'm not sure why `??|` works. Table [9.41 in the docs](https://www.postgresql.org/docs/9.4/functions-json.html) shows `?|`. There had to be a SO post that I pulled this syntax from.

Reply:
Oops, wrong link ... should be version 11: https://www.postgresql.org/docs/11/functions-json.html

Reply:
Aha!!! I remember now. The double `??` is an artifact of jdbi3: https://jdbi.org/#_postgresql where some operators need to be escaped.

Reply:
Oh I see. There's an operator collision for ```?``` so you have to use ```??``` instead. I was going to ask you about that next, but you've pretty much anticipated it.